### DISCUSSION TITLE
How to upload the image that is captured using Camera?

### DISCUSSION THREAD

**Discussion by ChaitanyaV98** [rpldy/react-uploady#214] [Q&A]
I am able to upload files from gallery/drive where I have used document picker to select the files and upload them using react-uploady. The other case I want to handle is to open a camera and upload the image for which I have used image-picker to launch camera but I am not able to achieve. Can anyone plz help me how to resolve this issue using native-uploady?

**Comment by yoavniran** [comment]
@ChaitanyaV98 what is the issue? what happens after the camera takes the photo?

**Reply by ChaitanyaV98** [reply]
When I take a picture using camera and click on ok, it it returning me a response object. the respone is in following format

`{
   "assets":[
      {
         "fileName":"rn_image_picker_lib_temp_ae3c8576-d676-4aad-995c-db94279bb9d1.jpg",
         "fileSize":1755346,
         "height":4160,
         "type":"image/jpeg",
         "uri":"file:///data/user/0/com.wiredup.global/cache/rn_image_picker_lib_temp_ae3c8576-d676-4aad-995c-db94279bb9d1.jpg",
         "width":3120
      }
   ]
}`

I have defined  res= response.assets

then passed to uploady like this: uploadyContext.upload(res);

But I am getting the error as -> Error: Unknown type of file added: object

**Reply by yoavniran** [reply]
Uploady expects that the object passed to it is either a File, Blob or an object with at least these props: 

```
{ name: String, type: String }
```

Please try converting the "fileName" property of the assets objects to "name" and see if that solves the problem.

**Reply by ChaitanyaV98** [reply]
Thanks @yoavniran.  But yeah I have tried this before, but the uri it is expecting is in conetent form but the response of image picker as u can see above is in file format.

**Reply by ChaitanyaV98** [reply]
If I pass only `name` and `type` params, still I am getting the upload error @yoavniran. Do we need not pass `uri` param?
